res/css/views/voip/_DialPad.scss

```diff
@@ -30,6 +30,7 @@ limitations under the License.
     text-align: center;
     vertical-align: middle;
     line-height: 40px;
+    color: #15191e;
```

Comment:
There might be a suitable colour for this in one of the theme files (e.g. `_dark.scss`). Could you please take a look?

Reply:
There are a few i could find but they're not properly labelled .
 
Also they light theme doesn't have these issues so it might get a bit messy , I think I could just update the `$theme-button-bg-color`  in _dark.css . 

So it becomes consistent with light theme. Something like this:
![image](https://user-images.githubusercontent.com/43457420/112285112-f73bf800-8caf-11eb-8a5a-b8bf168cff04.png)


So it becomes consistent(Like orignal Light theme)
![image](https://user-images.githubusercontent.com/43457420/112285030-e12e3780-8caf-11eb-86e5-33b3bf1490e6.png)

Reply:
Changing the value of `$theme-button-bg-color` isn't a good idea. The consistency would be great. If there really isn't anything suitable, you could perhaps add a new colour (both to `_light.scss` and `_dark.scss`)?

Reply:
I just added `$dialpad-button-bg-color` to all the themes(it wasn't compiling without all of them) for the case. I hope this would be alright